src/main/java/com/marklogic/kafka/connect/source/CsvQueryContextBuilder.java

```diff
@@ -14,27 +14,21 @@
 import java.util.List;
 import java.util.Map;
 
-public class CsvQueryContextBuilder extends AbstractRowBatchBuilder implements QueryContextBuilder<String> {
+public class CsvQueryContextBuilder extends AbstractRowBatchBuilder <String> {
```

Comment:
Not urgent - can be changed in a subsequent PR - but the subclasses should go back to "(Type)RowBatcherBuilder" and the parent class needs to have "Batch" changed to "Batcher".